cuda/base/cusparse_bindings.hpp

```diff
@@ -226,6 +265,8 @@ GKO_BIND_CUSPARSE64_SPMM(ValueType, detail::not_implemented);
 #undef GKO_BIND_CUSPARSE32_SPMM
 #undef GKO_BIND_CUSPARSE64_SPMM
 
+#endif
```

Comment:
We currently put the function which does similar thing together.
It needs many `#if`-block but easily identify which function has different version.
Another way is to create a global `#if` to handle these function.
It can reduce the number of `#if`-block but maybe gives very complicated structure.
For example, we need to handle several version of benchmark/spmv.
1. old coo/csr/ell/hybrid is available in 9 ~ 10.2
2. generic coo/csr/csr is available after 10.2 in non-Windows or 11 in all platform.
3. coo/csr is alias for gcoo/gcsr after 11

Reply:
I mean, the previous one in this case has the same definition I think, so it wouldn't hurt to simply merge it with the one above. You do not have to change the file structure.

Reply:
I see